src/Directory.Packages.props

```diff
@@ -23,7 +23,7 @@
     <PackageVersion Include="System.Threading.AccessControl" Version="5.0.0" Condition="'$(TargetFramework)' != 'net461'" />
     <PackageVersion Include="ZooKeeperNetEx" Version="3.4.12.4" />
     <PackageVersion Include="IsExternalInit" Version="1.0.3" />
-    <PackageVersion Include="Microsoft.Bcl.AsyncInterfaces" Version="5.0.0" Condition="'$(TargetFramework)' == 'netstandard2.0' OR '$(TargetFramework)' == 'net461'" />
+    <PackageVersion Include="Microsoft.Bcl.AsyncInterfaces" Version="8.0.0" Condition="'$(TargetFramework)' == 'netstandard2.0' OR '$(TargetFramework)' == 'net461'" />
```

Comment:
Open Question 2: There's this chain of transitive dependency: `Npgsql` 8.0.2 -> `System.Text.Json` 8.0.0 -> `Microsoft.Bcl.AsyncInterfaces` 8.0.0. However, we have `Microsoft.Bcl.AsyncInterfaces` 5.0.0 here, so we're getting `error NU1109: Detected package downgrade: Microsoft.Bcl.AsyncInterfaces from 8.0.0 to centrally defined 5.0.0. Update the centrally managed package version to a higher version.`. Therefore, I made this change, but now we see this warning: `warning : Microsoft.Bcl.AsyncInterfaces 8.0.0 doesn't support net461 and has not been tested with it. Consider upgrading your TargetFramework to net462 or later. You may also set <SuppressTfmSupportBuildWarnings>true</SuppressTfmSupportBuildWarnings> in the project file to ignore this warning and attempt to run in this unsupported configuration at your own risk. [DistributedLock\src\DistributedLock.ZooKeeper\DistributedLock.ZooKeeper.csproj::TargetFramework=net461]`
This brings us back to https://github.com/madelson/MedallionShell/issues/113; should we continue to support `net461` and other versions which already reached end of support?

Reply:
Let's go to net462 throughout and see if everything is happy. My main goal is to keep supporting .NET framework for the time being, and not to drop support for specific framework versions without a reason (but this _is_ a reason!).

Reply:
@madelson AppVeyor seems happy! 🎉 
Note that I didn't update any versions or release notes.